src/test/run-pass/imports/extern-crate-self-macro-1.rs

```diff
@@ -0,0 +1,11 @@
+fn the_answer() -> usize { 42 }
```

Comment:
Move this file to `ui/imports/` and explicitly write `// run-pass` on the first line.

If you can put all the test files related to `extern crate self` into `imports/extern-crate-self` that would be nice.